expression/schema.go

```diff
@@ -30,6 +30,14 @@ func (ki KeyInfo) Clone() KeyInfo {
 	return result
 }
 
+func (ki KeyInfo) String() string {
```

Comment:
Add a comment, please.

Reply:
Since we added this method, we can remove the duplicate code in `(*Schema).String()`.

Reply:
updated.